uncertainty_toolbox/utils.py

```diff
@@ -0,0 +1,20 @@
+import numpy as np
+
+
+def check_flat_same_shape(*args):
```

Comment:
My preference would be to have this return a boolean (rename to is_flat_same_shape) and then use assert when it is called in the code, but either way works. This would make it more clear when reading the code that if we don't have the correct shape we will throw an error.